src/package.json

```
@@ -43,6 +43,8 @@
     "node-fetch": "2.6.1",
     "xml-js": "1.6.11",
     "run-script-os": "1.1.3",
-    "node-watch": "0.7.0"
-  }
+    "node-watch": "0.7.0",
+    "rainbow-code": "^2.1.7"
```

We do hard versions and then dependabot notifies us of any updates. This gives us visibility of dependency updates (though on flip side also means we get an email and automatci PR when one does happen and know that annoys some people!)

```suggestion
    "rainbow-code": "2.1.7"
```